src/elf/dyn.rs

```diff
@@ -204,6 +245,22 @@ pub fn tag_to_str(tag: u64) -> &'static str {
         DT_VERNEED => "DT_VERNEED",
         DT_VERNEEDNUM => "DT_VERNEEDNUM",
         DT_FLAGS_1 => "DT_FLAGS_1",
+        DT_MIPS_RLD_VERSION => "DT_MIPS_RLD_VERSION",
+        DT_MIPS_TIME_STAMP => "DT_MIPS_TIME_STAMP",
+        DT_MIPS_ICHECKSUM => "DT_MIPS_ICHECKSUM",
+        DT_MIPS_IVERSION => "DT_MIPS_IVERSION",
+        DT_MIPS_FLAGS => "DT_MIPS_FLAGS",
+        DT_MIPS_BASE_ADDRESS => "DT_MIPS_BASE_ADDRESS",
+        DT_MIPS_CONFLICT => "DT_MIPS_CONFLICT",
+        DT_MIPS_LIBLIST => "DT_MIPS_LIBLIST",
+        DT_MIPS_LOCAL_GOTNO => "DT_MIPS_LOCAL_GOTNO",
+        DT_MIPS_CONFLICTNO => "DT_MIPS_CONFLICTNO",
+        DT_MIPS_LIBLISTNO => "DT_MIPS_LIBLISTNO",
+        DT_MIPS_SYMTABNO => "DT_MIPS_SYMTABNO",
+        DT_MIPS_UNREFEXTNO => "DT_MIPS_UNREFEXTNO",
+        DT_MIPS_GOTSYM => "DT_MIPS_GOTSYM",
+        DT_MIPS_HIPAGENO => "DT_MIPS_HIPAGENO",
+        DT_MIPS_RLD_MAP => "DT_MIPS_RLD_MAP",
```

Comment:
Since these are processor specific, I'm not sure it's correct to return them without knowing that the processor is MIPS. Is this something that other libraries do?

Reply:
Normally they're just `#define`. I don't think it should be an issue, the tags shouldn't overlap.

Reply:
`tag_to_str` is used in `<Dyn as Debug>::fmt`. Since we can't pass a processor argument to  `Debug::fmt`, I don't see any way for that to correctly print these values. So `tag_to_str` can't print these values. We could improve it to print something like `PROCESSOR_SPECIFIC` instead `UNKNOWN_TAG` though.

If there is a need to get these strings for other use cases, then you could define an additional function that does accept a processor argument.